asb-ballerina/tests/asb_test.bal

```diff
@@ -21,17 +21,30 @@ import ballerina/time;
 
 // Connection Configurations
 configurable string connectionString = os:getEnv("CONNECTION_STRING");
-configurable string queueName = os:getEnv("QUEUE_NAME");
-configurable string topicName = os:getEnv("TOPIC_NAME");
-configurable string subscriptionName1 = os:getEnv("SUBSCRIPTION_NAME1");
+configurable string queueName = "myqueue";
```

Comment:
These can be changed? right?

Reply:
yeah these are still configurables and just added the default values for convenience. these can still be overriden by the  values provided via a `config.toml`.